refact-agent/gui/src/hooks/useGetUser.ts

```diff
@@ -15,29 +16,31 @@ export const useGetUser = () => {
   const maybeAddressURL = useAppSelector(selectAddressURL);
   const addressURL = maybeAddressURL ? maybeAddressURL.trim() : "";
   const maybeApiKey = useAppSelector(selectApiKey);
+  const isStreaming = useAppSelector(selectIsStreaming);
   const apiKey = maybeApiKey ?? "";
   const isAddressURLALink =
     addressURL.startsWith("https://") || addressURL.startsWith("http://");
 
   const request = smallCloudApi.useGetUserQuery(
     { apiKey, addressURL: addressURL },
     {
-      skip: !(
-        NOT_SKIPPABLE_ADDRESS_URLS.includes(addressURL) || isAddressURLALink
-      ),
+      skip:
+        !(
+          NOT_SKIPPABLE_ADDRESS_URLS.includes(addressURL) || isAddressURLALink
+        ) || isStreaming,
       refetchOnMountOrArgChange: true,
     },
   );
 
   useEffect(() => {
```

Comment:
I'm seeing a lot of `setIntialAgentUsage` being called form here.

A way around this would be to add an `extreReducer` on `agentUsageSlice` that matches the fullfilled `getUser` call from `smallCloudApi`

```ts
  extraReducers:(builder) => {
    builder.addMatcher(smallCloudApi.endpoints.getUser.matchFulfilled, (state, action) => {
       // update logic here
    }
  }
```